packages/DatePicker/src/components/Calendar/Calendar.js

```diff
@@ -178,7 +178,7 @@ function Calendar(props) {
   }
 
   const CalendarKey = `${currentMonth && currentMonth.format("YYYY-MM")}/${possibleDate &&
-    possibleDate.format("YYYY-MM")}/${date && date.format("YYYY-MM")}/${isVisible}`;
+    possibleDate.format("YYYY-MM")}/${date && date.format("YYYY-MM")}`;
```

Comment:
@allison-c  Was there a particular reason you wanted the calendar to re-render every time someone opened the popup?

Reply:
I was trying to force `<SDPController />` re-render to run `getInitialVisibleMonth`.  However I tested your storybook, it works fine now..

Reply:
Oh I think I found the bug

1. Go to http://storybooks.highbond-s3.com/paprika/ux-242-date-picker-positioning/?path=/story/datepicker--datepicker
2. Use Calendar select Oct 30
3. Click Input again and keep clicking next month button
4. Click outside
4. Click Input again and see the Calendar

You can see the Calendar shows future but it should be October cause we didn't select any date in the second time